package.json

```diff
@@ -39,6 +39,8 @@
     "xo": "^0.24.0"
   },
   "scripts": {
+    "gulp": "gulp",
```

Comment:
Do we need this script? I don't think it's used 🤔

Reply:
it allows user not to install gulp globally by running `npm run gulp`

This command can be replaced by `npx gulp` anyway, and we have a start script, so yes it can be dropped

Reply:
Yeah, my main reasoning was we already have the start script 👍